### PR TITLE
Add report objects for manageability suite tests

### PR DESCRIPTION
**Pull request by sebrandon1**
build-depends: 28703
- Add compliant/nonCompliant objects for these two tests.
- Add Container.IsTagEmpty() func with corresponding unit tests.

### CODE REVIEW

File: cnf-certification-test/manageability/suite.go
Comment (edcdavid) [inline cnf-certification-test/manageability/suite.go:61]: no need to add the container id -> fmt.Sprintf("Container %s is missing image tag(s)", cut.String()) -> "Container is missing image tag(s)"

File: cnf-certification-test/manageability/suite.go
Comment (edcdavid) [inline cnf-certification-test/manageability/suite.go:89]: optional: consider re-using port/protocol fields defined in https://github.com/test-network-function/cnf-certification-test/pull/1207 and add 2 level report, one per container per port and one per pod

**Review by jmontesi — APPROVED** [review]
Approved waiting for updates after David's comments.

File: cnf-certification-test/manageability/suite.go
Comment (sebrandon1) [reply cnf-certification-test/manageability/suite.go:89]: I'll adjust this once the #1207 gets reviewed and merged.

File: cnf-certification-test/manageability/suite.go
Comment (sebrandon1) [reply cnf-certification-test/manageability/suite.go]: Fixed.

**Review by greyerof — APPROVED** [review]

**Pull request merged**
